Выполнил дз номер 1

AntonFedorovich95 opened the pull request.

AntonFedorovich95 committed:
Выполнил дз номер 1

OlgaLugacheva commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,58 @@
+        long d = 99L;
+        float e = 20.75f;
+        double f = 77.98;
+
```
char и boolean забыты

OlgaLugacheva commented on src/ru/skypro/Main.java:
```diff
@@ -3,6 +3,58 @@
+        System.out.println("Маша теперь получает " + newsalaryMasha + " рублей." + " Годовой доход вырос на " + incomedifferenceMasha + " рублей.");
+        System.out.println("Денис теперь получает " + newsalaryDenis + " рублей." + " Годовой доход вырос на " + incomedifferenceDenis + " рублей.");
+        System.out.println("Кристина теперь получает " + newsalaryKris + " рублей." + " Годовой доход вырос на " + incomedifferenceKris + " рублей.");
+
+
+
+
```
Слишком много пустых строк

AntonFedorovich95 committed:
Добавил переменные char в 2ух вариантах исполнения.
Добавил переменную boolean в значение true, при смене знака будет false.
Оставил по 1 пустой строке между заданиями.
Выполнил дополнительные задания.

AntonFedorovich95 committed:
Merge pull request #2 from AntonFedorovich95/home2.2

Добавил переменные char в 2ух вариантах исполнения.
